app/src/main/java/com/sun/music_64/screen/home/HomeFragment.java

```diff
@@ -13,8 +15,19 @@
 import android.widget.TextView;
 
 import com.sun.music_64.R;
+import com.sun.music_64.data.model.Track;
+import com.sun.music_64.screen.genrescreen.GenreActivity;
+import com.sun.music_64.screen.genrescreen.GenreEntity;
+import com.sun.music_64.screen.genrescreen.ItemClickRecyclerView;
 
-public class HomeFragment extends Fragment {
+import java.util.ArrayList;
+import java.util.List;
+
+public class HomeFragment extends Fragment implements View.OnClickListener, ItemClickRecyclerView
+        , SuggestContract.View {
+
+    private static final int LIMIT = 20;
```

Comment:
Tất cả các hằng số cần comment về ý nghĩa của chúng
```suggestion
    /**
    * INSERT HERE
    **/
    private static final int LIMIT = 20;
```

Reply:
![new](https://user-images.githubusercontent.com/48239173/58153417-2428b980-7c99-11e9-9091-372ccf4da1f5.png)

em sua roi a a